workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -47,7 +55,7 @@ process alevin{
     path run_dir
   script:
     // label the run directory by id, index, and mapping mode
-    run_dir = "${id}-${index}-${params.sketch ? 'sketch' : 'salign'}-${params.unfiltered ? 'unfiltered' : 'knee'}"
+    run_dir = "${id}-${index}-${params.sketch ? 'sketch' : 'salign'}-${params.resolution}-${params.filter == 'knee' ? 'knee' : ''"}"
```

Comment:
I flipped the logic here to keep make this blank when unfiltered and add the filter name (and dash) only when it is something else. 

```suggestion
    run_dir = "${id}-${index}-${params.sketch ? 'sketch' : 'salign'}-${params.resolution}${params.filter == 'unfiltered' ? '' : "-${params.filter}" }"
```
For a bit more clarity (and shorter line length), we could break this up:
```suggestion
    run_dir = "${id}-${index}-${params.sketch ? 'sketch' : 'salign'}-${params.resolution}"
    if(params.filter != 'unfiltered'){
      run_dir += "-${params.filter}"
    }
```